tests/unit/plugins/module_utils/test_client.py

```diff
@@ -71,8 +71,18 @@ def test_json_is_cached(self, mocker):
 
 
 class TestClientAuthHeader:
+    def test_basic_auth_no_host(self):
+        msg = "Invalid instance host value: ''. Value must start with 'https://' or 'http://'"
+        with pytest.raises(errors.ServiceNowError, match=msg):
+            client.Client("", "user", "pass")
+
+    def test_basic_auth_host_without_schema(self):
+        msg = "Invalid instance host value: 'instance.com'. Value must start with 'https://' or 'http://'"
+        with pytest.raises(errors.ServiceNowError, match=msg):
+            client.Client("instance.com", "user", "pass")
```

Comment:
We are testing the constructor, not the `Client.auth_header` property with these, so those two tests should live in a dedicated `TestClientInit` class.

Reply:
OK.